spec/system/admin/instrument_price_policies_controller_spec.rb

```diff
@@ -400,4 +402,105 @@
       expect(page).not_to have_content("$90.00") # Step 3 rate (removed)
     end
   end
+
+  context "Schedule Rule Daily pricing mode" do
+    let(:pricing_mode) { Instrument::Pricing::SCHEDULE_DAILY }
+
+    it "can add price policies", :js do
+      visit new_facility_instrument_price_policy_path(facility, instrument)
+
+      expect(page).to have_content("Rate Per Day")
+      expect(page).to_not have_content("Rate Per Hour")
+
+      expect(page).to_not have_content("Reservation Cost")
+      expect(page).to_not have_content("Minimum Cost")
+
+      PriceGroup.find_each do |price_group|
+        expect(page).to_not have_field("price_policy_#{price_group.id}[minimum_cost]")
+        expect(page).to_not have_field("price_policy_#{price_group.id}[cancellation]")
+        expect(page).to_not have_field("price_policy_#{price_group.id}[usage_rate]")
+      end
+
+      fill_in("note", with: "Some note about pricing")
+
+      check("price_policy_#{base_price_group.id}[can_purchase]")
+      check("price_policy_#{cancer_center.id}[can_purchase]")
+      uncheck("price_policy_#{external_price_group.id}[can_purchase]")
+
+      fill_in("price_policy_#{base_price_group.id}[usage_rate_daily]", with: "100")
+      fill_in("price_policy_#{cancer_center.id}[usage_subsidy_daily]", with: "30")
+
+      find_field(
+        "price_policy_#{cancer_center.id}[usage_rate_daily]",
+        type: "hidden"
+      ).tap do |hidden_input|
+        expect(hidden_input.value).to eq("100")
+      end
+
+      expect(page).to_not have_content("/ minute")
+
+      expect { click_button("Add Pricing Rules") }.to(
+        change { instrument.price_policies.reload.count }.by(3)
+      )
+
+      expect(page).to(
+        have_current_path(facility_instrument_price_policies_path(facility, instrument))
+      )
+      expect(page).to have_content("Price Rules were successfully created.")
+    end
+
+    describe "already created price policies" do
+      let(:start_date) { Date.today }
+      let(:expire_date) { PricePolicy.generate_expire_date(@start_date) }
+      let(:base_price_policy) { instrument.price_policies.find_by(price_group: base_price_group) }
+      let!(:price_policies) do
+        PricePolicyBuilder.get_new_policies_based_on_most_recent(
+          instrument, start_date
+        ).each do |price_policy|
+          price_policy.assign_attributes(
+            note: "Some note about pricing",
+            usage_rate_daily: 100,
+            charge_for: InstrumentPricePolicy::CHARGE_FOR[:reservation]
+          )
+
+          price_group = price_policy.price_group
+          price_policy.usage_subsidy_daily = 10 if price_group.external? || price_group.master_internal?
+
+          price_policy.save!
+        end
```

Comment:
I think you should be able to create the Price policy using the factory

Reply:
makes sense